File: 백준/Bronze/3059. 등장하지 않는 문자의 합/등장하지 않는 문자의 합.py
import sys
input = sys.stdin.readline

t = int(input().rstrip())
for _ in range(t):
    string = input().rstrip()
    alpha = {chr(k+ord('A')):0 for k in range(26)}
    for s in string:
        alpha[s] = 1
    
    result = 0
    for k in range(26):
        if alpha[chr(k+ord('A'))] == 0:
            result += k + 65
    
    print(result)

File: 백준/Bronze/3059. 등장하지 않는 문자의 합/등장하지 않는 문자의 합.cc
#include <stdio.h>
#include <string.h>

int main(void) {
    int t;
    scanf("%d", &t);

    char str[1005];
    while(t--) {
        scanf("%s", str);

        int alpha[26] = {0,};
        for (int i = 0; i < strlen(str); i++) alpha[str[i] - 'A'] = 1;

        int result = 0;
        for (int i = 0; i < 26; i++) {
            if (alpha[i] == 0) result += i + 'A';
        }

        printf("%d\n", result);
    }
    return 0;
}